### PR TITLE
remove old_div in koala_rss, convert words to english, small reformating 

### PR DESCRIPTION
**Pull request by YourLocalBlake**

### CODE REVIEW

File: src/koala/__init__.py [hunk 1]
```diff
@@ -25,8 +26,8 @@
 
 import numpy as np
 
-from synphot import observation
-from synphot import spectrum
```
Comment (YourLocalBlake) [inline src/koala/__init__.py]: whoops, forgot to convert this back - I run pysynphot locally

**Comment by YourLocalBlake** [comment]
codecov/project can fail? @aragilar

**Comment by aragilar** [comment]
Yes, as the coverage has gone down (likely due to adding back the functions).

**Comment by YourLocalBlake** [comment]
Ah okay, I thought that would just result in a warning of some kind

**Comment by aragilar** [comment]
The level of drop can be configured, but it's preferable to fail, as likely the reason is due to missing tests

**Comment by YourLocalBlake** [comment]
ah, so for the moment it's probably best just to raise a warning while lots of code is still being moved around?

File: src/koala/__init__.py [hunk 2]
```diff
@@ -572,11 +572,13 @@ def correct_high_cosmics_and_defects(
 
         if plot_suspicious_fibres == True and len(suspicious_fibres) > 0:
             # Plotting suspicious fibres..
-            figures = plot_suspicious_fibres(suspicious_fibres,
-                                   fig_size,
-                                   wave_min,
-                                   wave_max,
-                                   intensity_corrected_fiber=self.intensity_corrected)
+            figures = plot_suspicious_fibres_graph(
```
Comment (YourLocalBlake) [inline src/koala/__init__.py]: This change references #15 where plot_suspicious_fibres was introduced as a plotting function, overwriting the bool option

**Pull request merged**